docs/gettingStarted/install.rst

```diff
@@ -193,14 +193,26 @@ Preparing your Azure environment
 
 Follow the steps below to prepare your Azure environment for running a Toil workflow.
 
-#. Create an `Azure account`_.
+#. Create an `Azure account`_ and to use the job store make an `Azure storage account`_.
```

Comment:
I feel like the cloud account setup for `aws` and `azure` should be in their respective sections and not in the installation section.  Is there a good reason to keep this here?

Reply:
I was just following the example of what was there before. I'm indifferent on moving it.

Reply:
Alright, it seems fine to leave it for a separate PR/discussion then.